2.5.2/src/Merchandise.java

```diff
@@ -0,0 +1,59 @@
+public class Merchandise {
+    private String type;
+    private String name;
+    private double price;
+    private int quantity;
+
+    public Merchandise(String type, String name, int quantity, double price) {
+        this.type = type;
+        this.name = name;
+        this.quantity = quantity;
+        this.price = price;
+    }
+
+    public String getType() {
+        return type;
+    }
+
+    public void setType(String type) {
+        this.type = type;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public double getPrice() {
+        return price;
```

Comment:
Could u explain why at different places u code different ways for `get/set`, please?
I mean next:
```
public String getName() {
    return name;
}

public void setType(String type) {
    this.type = type;
}
```